flow/connectors/eventhub/scoped_eventhub.go

```diff
@@ -16,16 +16,17 @@ type ScopedEventhub struct {
 }
 
 func NewScopedEventhub(dstTableName string) (ScopedEventhub, error) {
-	// split by dot, the model is peername.eventhub.partition_key_column
+	// split by dot, the model is peername.eventhub.uniquekey.partition_key_column
```

Comment:
why is `uniquekey` required? we can keep backwards compatibility by accepting previous a.b.c notation

Reply:
This could be addressed by adding scripting eventhubs, where scripts can generate N messages for M topics